data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,401 @@ mission "Remnant: Face to Maw 2B"
 			scene "outfit/void rifle"
+
+
+
+mission "Remnant: Crystal Research 1"
+	name "Retrieve more Crystals"
+	description `Retrieve Crystals from Greenwater, and then Allhome.`
+	minor
+	source "Viminal"
+	destination "Greenwater"
+	cargo "(reserved)" 2
+	to offer
+		random < 30
+		has "Remnant: Learn Sign 2: done"
+		has "Remnant: Technology Available: offered"
+	on offer
+		log "People" "Aeolus" `As an outfitter, Aeolus is responsible for managing the logistics that supply the Remnant with materials, equipment, and resources. In contrast to outfitters in what the Remnant term 'ancestral space,' he is focused on ensuring the efficient optimization of Remnant ships, not making a profit.`
+		conversation
+			`In the spaceport cafeteria you spot the outfitter who had you retrieve a shipment of Hai Keystones. He notices you and waves you over, gesturing at an empty chair opposite him.`
+			`	"Greetings, <first>! I trust you are settling in among us?" he signs using simple gestures. "I picked up a rumor that you are learning our language, too." You sign the affirmative as you sit down. "Good, that is useful." He pauses, then continues, "I never introduced myself before. My name is Aeolus, and I help manage resource logistics for our fleets." He eyes you for a moment, then ventures, "you came in here looking like you were watching for something. Anything on your mind?"`
+			choice
+				`	(Ask how business is going)`
+				`	(Ask if there are more jobs like the last one you did for him.)`
+					goto crystaljobs
+			`	"Well, fairly well. Stocks are up, there are quite a few new outfits in the current, and some recent supply line disruptions have been resolved." He gestures appreciatively. "I am told that we have you to thank for some of that.`
+			`	"Those crystals you brought back have been quite interesting. The process of dismantling them for analysis yielded some interesting results." He notices your questioning expression. "Oh, you were expecting me to sell them, weren't you?"`
+			choice
+				`	(Nod affirmatively.)`
+				`	(Shrug noncommittally.)`
+			`	He looks slightly embarrassed. "Sorry to disappoint you, but none of those outfits were ever going to be used in the form they came in, for several different reasons." He ticks them off on his fingers as he lists them. "First, security: Slapping an alien device on every ship without having thoroughly dismantled them would be an invitation to being tracked. Second, research: Something that does what the Quantum Key Stone does without being identical to it? Definitely interesting to our researchers. And third, purpose."`
```

Comment:
(Still learning with Github, couldn't figure out how to respond to your reply about the two "being"s, so this is my reply to that.)
`would be an invitation to being tracked`: `be` and `being` are used close together in this spot, which is why I suggested changing `being` to `getting`. Not a huge issue since they're only the same root word, not the exact same word.